reoptjl/models.py

```diff
@@ -1707,6 +1746,23 @@ class ElectricUtilityInputs(BaseModel, models.Model):
         default=list, blank=True,
         help_text=("CO2 emissions factor over all hours in one year. Can be provided as either a single constant fraction that will be applied across all timesteps, or an annual timeseries array at an hourly (8,760 samples), 30 minute (17,520 samples), or 15 minute (35,040 samples) resolution.")
     )
+    emissions_factor_CO2_decrease_fraction = models.FloatField(
+        validators=[
+            MinValueValidator(-3),
+            MaxValueValidator(3)
```

Comment:
is there a reason this changed to [-3,+3] from [-1,+1]?  I suppose it's possible for the emissions to more than double annually, but a value of >1 would yield negative emissions - is that something we'd consider?

Reply:
I think I may have confused this for the emissions reduction constraint input! I changed back to -1 and +1 in https://github.com/NREL/REopt_API/pull/475/commits/4196459ed9824beac569ded26138ebdcf3a91e8d